lib/auth/permissions.go

```diff
@@ -419,10 +429,11 @@ func (a *authorizer) authorizeRemoteBuiltinRole(r RemoteBuiltinRole) (*Context,
 		AllowedResourceIDs: nil,
 	}, a.clusterName, roleSet)
 	return &Context{
-		User:             user,
-		Checker:          checker,
-		Identity:         r,
-		UnmappedIdentity: r,
+		User:                       user,
+		Checker:                    checker,
+		Identity:                   r,
+		UnmappedIdentity:           r,
+		disableDeviceAuthorization: a.disableDeviceAuthorization,
```

Comment:
remote builtin roles don't skip device trust, but local ones (below) do?

Reply:
Not a great answer, but I was following suite from existing logic in `auth.Context.GetAccessState`: https://github.com/gravitational/teleport/blob/3a0fc94aa6e0605965ab853500e5bf3b7fdb7472/lib/auth/permissions.go#L189-L190

I'm not sure whether RemoteBuiltinRole is omitted on purpose there of if it's an overlook.